src/Directory.Build.targets

```diff
@@ -54,15 +54,6 @@
   <Target Name="GetDocumentationFile"
           Returns="$(DocumentationFile)"/>
 
-  <!-- This target will be moved into buildtools. Checked in here for now to add validationand in order not to block buildtools ingestion. -->
-  <Target Name="ValidateExcludeCompileDesktop" AfterTargets="GetPackageDependencies" Inputs="%(Dependency.Identity);%(Dependency.TargetFramework)" Outputs="unused">
-    <PropertyGroup>
-      <_excludeCompile Condition="@(Dependency->WithMetadataValue('Exclude', 'Compile')->Count()) == @(Dependency->Count())">true</_excludeCompile>
-    </PropertyGroup>
-    <Error Text="Cannot have Exclude=Compile dependencies when targeting a desktop TFM. @(Dependency). You can exclude the reference asset in the package by setting the ExcludeReferenceAssets property to true in your project." 
-           Condition="$([System.String]::Copy('%(Dependency.TargetFramework)').StartsWith('net4')) AND 
-                      '$(_excludeCompile)' == 'true' AND
-                      '%(Dependency.Identity)' != '_._'" />
-  </Target>
+  <Import Project="..\eng\Packaging.targets" />
```

Comment:
nit: can you put this in the root Directory.Build.targets as `<Import Project="$(RepositoryEngineeringDir)Packaging.targets" />` so that it parallels what we do here: 
--
https://github.com/dotnet/corefx/blob/375ba8f48581e36b02ee99691f55380bb42557c5/Directory.Build.props#L208

Reply:
Sure, it does mean that we are going to run these targets now on the pkgprojs under root/pkg folder but I guess that is fine.

Reply:
Yeah, I feel better about having all pkgproj's behave the same rather than have differences based on which folder they build from.